committee.rst

```diff
@@ -0,0 +1,152 @@
+way.
+
+Method of Communication
+-----------------------
+
+Most official committee business takes place via the
+`ghc-steering-commitee <https://mail.haskell.org/mailman/listinfo/ghc-steering-committee>`_ mailing list. The archives of this list are public, and
+any member of the community may subscribe. It is intended that the list
+be used by the committee, though community members might be asked
+to post there during relevant conversations.
+
+Any business not on the public mailing list will be of a sensitive nature,
+most likely pertaining to individuals. In particular, discussions of selecting
+new members and of violations of the guidelines for respectful communication
+will not appear on the mailing list.
+
+Term Limits and Committee Selection
+-----------------------------------
+
+Any community thrives best by continuing to refresh itself with new members.
+Accordingly, membership on the committee comes with a three-year term. Terms
+begin/end in October. Toward the end of an individual's term, the secretary of
+the committee will put out a public call for nominations for people to join
+the committee. Self-nominations are welcome and expected. The member whose
+term is ending is free to re-nominate themselves. Nominations include a brief
+bio of the nominee and reasons why they would be appropriate for the
+committee.
+
+Continuing members of the committee vote on new membership via a ranked voting
+system; the precise details (i.e. election algorithm) are recommended by the
+secretary but can be vetoed by a majority vote. The ranking system includes a
+method for choosing multiple winners. It is expected that the committee
+favor newcomers in preference to re-nominated continuing members. If
+an expiring member continues, the secretary shall ensure that a
+rationale for this decision be published to the community.
+
+The voting process may result in a number of new members not equal to
+the number of outgoing members. This is fine; the size of the committee
+is not fixed. However, a new nomination process begins only when
+membership drops below nine members.
+
+The nomination and voting process is kept private, by using direct
+email to committee members, not the mailing list. When the secretary's
+term is expiring (but that individual wishes to stay on the committee),
+one co-chair fills in for the secretary to run this process.
+
+Any member of the committee is free to step down at any time. Doing so
+causes a nomination process to begin (if the resulting size of the committee
+would be less than nine).
+
+There are two exceptions to the three-year term rule for two "key members"
+of the committee: in recognition of
+their historical and current importance to GHC, both Simon Peyton Jones
+and Simon Marlow will be expected to be retained on the committee when
+their terms end. If either wishes to continue serving on the committee
+when their terms end, and with the support of the majority of the committee,
```

Comment:
terms end, and if the majority of the rest of the committee supports this outcome, no public nomination process...